static/js/drafts.js

```diff
@@ -153,18 +153,18 @@ function draft_notify() {
 
 export function update_draft() {
     const draft = snapshot_message();
+    const draft_id = $("#compose-textarea").data("draft-id");
 
     if (draft === undefined) {
```

Comment:
Can you confirm that this is the value `draft` will have after clearing the compose box?  It's not obvious to me that's the case.

Reply:
Yes, the value of `draft` will be `undefined` because the length of text will be less than 2, https://github.com/zulip/zulip/blob/0f3d45f0786b8a047b650773249b3d9f37a431ed/static/js/drafts.js#L93

But it raises a condition where it deletes the drafts if the length of the text is <= 2.

Should we save the draft even if the message content is <=2 and > 0? (I'll ask it in CZO)

Reply:
https://chat.zulip.org/#narrow/stream/101-design/topic/saved.20as.20draft/near/1200221

Reply:
@timabbott This PR is read for review as the current changes follow what we have [decided](https://chat.zulip.org/#narrow/stream/101-design/topic/saved.20as.20draft/near/1200298).